src/style.css

```diff
@@ -1,3 +1,67 @@
 body {
   margin: 0;
+  --nav-hover-color: #00acdc;
+}
+
+header {
+  display: flex;
+  align-items: center;
+  justify-content: space-between;
+  background-color: white;
+  padding: 0 50px;
+}
+
+.logo {
+  height: 40px;
+  width: 40px;
+}
+
+.nav__header {
+  height: 60px;
+}
```

Comment:
no needs to hardcode height for elements with content inside. add padding to `nav__link`
```suggestion
```